src/bioregistry/data/bioregistry.json

```diff
@@ -30627,6 +30627,41 @@
     "pattern": "^\\d{7}$",
     "uri_format": "http://purl.obolibrary.org/obo/ECAO_$1"
   },
+  "eccdna": {
+    "contact": {
+      "email": "zhiyunguo@swjtu.edu.cn",
+      "name": "Zhiyun Guo",
+      "orcid": "0000-0003-2680-785X"
+    },
+    "contributor": {
+      "email": "palayoor.n@northeastern.edu",
+      "github": "nalikapalayoor",
+      "name": "Nalika Palayoor",
+      "orcid": "0009-0008-8406-631X"
+    },
+    "description": "scEccDNAdb is a database of single cell eccDNA entries of human and mouse samples, each with a unique identifier and features such as oncogenes, enhancers, and structural variants.",
+    "example": "ecDNA_hsa_01",
+    "github_request_issue": 1366,
+    "homepage": "https://lcbb.swjtu.edu.cn/scEccDNAdb/",
+    "name": "Single Cell ECC DNA Database",
+    "pattern": "^ecDNA_[a-z]+_\\d+$",
```

Comment:
it's surprising that the resource is called `eccdna` but the identifier has `ecDNA` with one fewer `c`

Reply:
Actually, I realize now that it's even more complicated. The example ID is valid 
![image](https://github.com/user-attachments/assets/1fcbfff6-451e-4dde-ae43-9c8ce59d7435)

but looking further, there are actually other ID's that use ecc as a prefix:

![image](https://github.com/user-attachments/assets/0e6b69da-680c-4f15-840b-9c6ce2108624)

😮 

So the regex pattern will need to be slightly adjusted and we can add an extra example.

Reply:
For future reference, there are two types of entries: ecDNA (extrachromosomal DNA) and eccDNA (extrachromosomal circular DNA) so the ID pattern makes sense. I added an extra example to show both types.

Reply:
There is a problem: for the `eccDNA_*` case:
```
            for example in entry.example_extras:
                with self.subTest(prefix=prefix, identifier=example):
>                   self.assertEqual(entry.standardize_identifier(example), example)
E                   AssertionError: 'hsa_35832' != 'eccDNA_hsa_35832'
E                   - hsa_35832
E                   + eccDNA_hsa_35832
```
This has to do with some core code in `standardize_identifier` that automatically strips off the prefix if followed by a `_` at https://github.com/biopragmatics/bioregistry/blob/36cad81452322bf9fffb16e307662db3ce7fbc56/src/bioregistry/schema/struct.py#L1965-L1966. This might have to be changed since in this case we wouldn't want to apply it.

Reply:
Set `"banana_peel": "",` to get around this

Reply:
Unfortunately `_` is hard-coded as an additional potential banana peel here:
https://github.com/biopragmatics/bioregistry/blob/36cad81452322bf9fffb16e307662db3ce7fbc56/src/bioregistry/schema/struct.py#L1960 so that won't work by itself.